app/src/main/res/values-en/strings.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="app_name">Practice</string>
```

Comment:
It's not necessary to defined a *-en* folder, you can use these strings from default strings.